connection.go

```diff
@@ -63,7 +76,14 @@ func (c *conn) IsValid() bool {
 	return c.session.GetStatus().StatusCode == cli_service.TStatusCode_SUCCESS_STATUS
 }
 
+// ExecContext executes a query that doesn't return rows, such
+// as an INSERT or UPDATE.
+//
+// ExecContext honors the context timeout and return when it is canceled.
 func (c *conn) ExecContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Result, error) {
+	if len(args) > 0 {
+		return nil, fmt.Errorf("databricks: query parameters are not supported")
+	}
 	_, opStatusResp, err := c.runQuery(ctx, query, args)
```

Comment:
Is there a difference between `cli_service.TExecuteStatementResp.GetModifiedRowCount()` vs `cli_service.TGetOperationStatusResp.GetNumModifiedRows()`? I was using the former in my implementation

Reply:
haha, good question. Can you check and submit a PR if this needs to be changed?